CONTRIBUTING.md

```diff
@@ -23,3 +23,29 @@ If you'd like to help make RSpec better, here are some ways you can contribute:
 If you need help getting started, check out the [DEVELOPMENT](DEVELOPMENT.md) file for steps that will get you up and running.
 
 Thanks for helping us make RSpec better!
+
+## `Small` issues
+
+These issues are the ones that we be believe are best suited for new
+contributors to get started on. They represent a potential meaningful
+contribution to the project that should not be too hard to pull off.
+
+## `Triage` issues
```

Comment:
maybe this should be "need reproduction"- `triage` is a label i've seen on many repos as `triaged` which flags that it has *already* been triaged, and as a result may have the opposite effect.